src/CacheInterface.php

```diff
@@ -64,7 +64,7 @@ public function clear();
      *   MUST be thrown if $keys is neither an array nor a Traversable,
      *   or if any of the $keys are not a legal value.
      */
-    public function getMultiple($keys, $default = null);
+    public function getMultiple(iterable $keys, mixed $default = null): iterable;
```

Comment:
(Can't comment on the correct line above this; thanks GitHub...)

Should the docblock be updated to say `iterable<string>`, or whatever the de facto convention is for that?  The code cannot be, yet, but the docblock could.  I don't know if that's scope creep or not, but worth considering.  (Here and in a few other methods.)